sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/RenderServiceVersion.java

```diff
@@ -0,0 +1,38 @@
+package com.azure.maps.render;
+
+import com.azure.core.util.ServiceVersion;
+
+public enum RenderServiceVersion implements ServiceVersion{
+     /**
+     * Service version {@code 1.0}.
```

Comment:
All teh versions in the swagger need to be here. Look at the azure communication SDK for a reference. this should return 2.1 for latest